<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="x0Input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x0Input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y0Input_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lambdaToleranceInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0.005</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>StartButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>toleranceInput</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>fileNameInput</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameInput">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>file_1.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="y0Input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y0Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lambdaToleranceInput</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="toleranceInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0.005</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>220</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="chartView" native="true">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>227</y>
      <width>671</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер функции (1 или 2)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="funcNumber">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
